tests/test_weighted_pandas.py

```diff
@@ -465,3 +466,18 @@ def test_WeightedSeries_nan():
     assert_allclose(series.mean(), 0.5, atol=1e-2)
     assert_allclose(series.var(), 1./12, atol=1e-2)
     assert_allclose(series.std(), (1./12)**0.5, atol=1e-2)
+
+
+def test_multiindex():
+    np.random.rand(0)
+    N = 1000
+    index_1 = np.arange(N)
+    index_2 = np.random.randint(0, 1000, N)
+    index = MultiIndex.from_arrays([index_1, index_2], names=['A', 'B'])
+    data = np.random.rand(N, 5)
+    weights = np.random.rand(N)
+    wdf = WeightedDataFrame(data, weights=weights, index=index)
+
+    assert wdf.index.names == ['A', 'B', 'weights']
+    assert_allclose(np.array([*wdf.index]).T, [index_1, index_2, weights])
+    assert wdf.reset_index().index.names == [None, 'weights']
```

Comment:
`reset_index` is only tested for its default arguments `level=None, drop=False, inplace=False`, here. Can we test the alternative arguments, too?

Reply:
Now added!